ArchiSteamFarm/IPC/WebUtilities.cs

```diff
@@ -32,6 +33,23 @@
 
 namespace ArchiSteamFarm.IPC {
 	internal static class WebUtilities {
+#if NETFRAMEWORK
+		internal static IMvcBuilder AddControllers(this IServiceCollection services) {
+			if (services == null) {
+				throw new ArgumentNullException(nameof(services));
+			}
+
+			return services.AddMvcCore();
+		}
+
+		internal static IServiceCollection AddRequestLocalization(this IServiceCollection services, Action<RequestLocalizationOptions> action) {
+			if (services == null) {
+				throw new ArgumentNullException(nameof(services));
+			}
+
+			return services.Configure(action);
+		}
+#endif
```

Comment:
Those look like they should be part of `Madness`.

Reply:
I wasn't sure whether we want AspNet stuff in Madness. Do we?

Reply:
I already have some asp.net there, so yes.

Reply:
merge request is up over there.